internal/charmstore/server_test.go

```diff
@@ -91,43 +92,68 @@ func (s *ServerSuite) TestNewServerWithVersions(c *gc.C) {
 }
 
 func (s *ServerSuite) TestNewServerWithConfig(c *gc.C) {
+
+	params := ServerParams{
+		AuthUsername:   "test-user",
+		AuthPassword:   "test-password",
+		IdentityAPIURL: "http://0.1.2.3/",
+	}
 	serveConfig := func(p *Pool, config ServerParams, _ string) HTTPCloseHandler {
 		return nopCloseHandler{
 			router.HandleJSON(func(_ http.Header, req *http.Request) (interface{}, error) {
 				return config, nil
 			}),
 		}
 	}
-	h, err := NewServer(s.Session.DB("foo"), nil, serverParams, map[string]NewAPIHandlerFunc{
+	h, err := NewServer(s.Session.DB("foo"), nil, params, map[string]NewAPIHandlerFunc{
 		"version1": serveConfig,
 	})
 	c.Assert(err, gc.IsNil)
 	defer h.Close()
+
+	// The IdentityLocation field is filled out from the IdentityAPIURL
+	// and the final slash is trimmed.
+
```

Comment:
Perhaps remove this blank line, the comment seems to be unrelated to anything here.